src/math_table/constants.rs

```diff
@@ -0,0 +1,91 @@
+use truetype::{Result, Tape, Value};
+
+table! {
+    #[doc = "math constants"]
+    pub Constants {
+        script_percent_scale_down (Percentage),
```

Comment:
Let’s keep the styling of `table!` consistent across the crate. Have a look at the alignment used in other tables.